src/module/actor/sheet/loot/loot-npcs-popup.ts

```diff
@@ -34,14 +36,51 @@ export class LootNPCsPopup extends FormApplication<ActorPF2e> {
             }
             const currentSource = token.actor;
             if (selectionData[i] && currentSource) {
-                const currentSourceItemData = currentSource.inventory.map((item) => item.toObject());
-                itemData.push(...currentSourceItemData);
-                const idsToDelete = currentSourceItemData.map((item) => item._id);
+                for (const item of currentSource.inventory) {
+                    const stackableItem = currentSource.findStackableItem(this.object, item);
```

Comment:
Should probably be this.object.findStackableItem() to keep it consistent with other uses of that same function.

After this I'm probably going to move findStackableItem() to actor.inventory and remove the first parameter, and add a deprecation warning for the existing function.

Reply:
yeah, I was wondering why wasn't at least a static method. I'll change it to this.object for now.